hmt_escrow/storage.py

```diff
@@ -196,7 +201,12 @@ def _encrypt(public_key: bytes, msg: str) -> bytes:
     """
     pub_key = keys.PublicKey(codecs.decode(public_key, "hex"))
     msg_bytes = msg.encode("utf-8")
-    return crypto.encrypt(msg_bytes, pub_key, shared_mac_data=SHARED_MAC_DATA)
+    shared_mac_data = (
+        SHARED_MAC_DATA
+        if isinstance(SHARED_MAC_DATA, bytes)
+        else SHARED_MAC_DATA.encode("ascii")
+    )
```

Comment:
@spirit-q2 can this be done at the top of the file, so it can be reused here and in the reference above w/o the repetition?

Reply:
found nice solution :slightly_smiling_face:

Reply:
seems to be working :tada: